### PR TITLE
Adds nextjs v13 working example

### PR DESCRIPTION
**Pull request by taylorjdawson**
### Description
Adds a working nextjs v13 example with web3-onboard and various wallets.

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] I tested locally to make sure this feature/fix works
- [x] I have run `yarn file-check`, `yarn type-check` & `yarn build` to confirm there are not any associated errors
- [x] This PR passes the Circle CI checks

### CODE REVIEW

File: examples/with-nextjs-13/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```
Comment (Adamj1232) [inline examples/with-nextjs-13/README.md:1]: May be helpful to add some context of where devs using this example should direct their attention in the example

File: examples/with-nextjs-13/web3-onboard.ts [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    dcent,
+    mew,
+    talltHoWalletSdk,
+    web3auth,
+    sequence
+  ],
+  // An array of Chains that your app supports
+  chains: [
+    {
+      // hex encoded string, eg '0x1' for Ethereum Mainnet
```
Comment (Adamj1232) [inline examples/with-nextjs-13/web3-onboard.ts:97]: Love all the explanation comments in here, great for examples!

File: examples/with-nextjs-13/app.d.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+declare module '@web3-onboard/tallyho'
```
Comment (Adamj1232) [inline examples/with-nextjs-13/app.d.ts:1]: @taylorjdawson Out of curiosity, what does this do and can we fix it at the package level?

**Review by Adamj1232 — APPROVED** [review]
Approved! Nice work! A couple questions and notes 🤓

File: examples/with-nextjs-13/app.d.ts (hunk 3 above)
Comment (taylorjdawson) [reply examples/with-nextjs-13/app.d.ts:1]: Actually I think the error may be gone now I am no longer seeing it 🤔

**Pull request merged**